examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<p>Can we provide an estimate of usage fees ("approximately $x.xx at time of writing of this notebook")? Should be ~$1-2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>